kit/dapp/src/components/dashboard/asset-manager-tile/asset-breakdown-donut.tsx

```
@@ -1,44 +1,80 @@
-  config,
-}: AssetBreakdownDonutProps) {
-  if (data.length === 0) {
+export function AssetBreakdownDonut({ data }: AssetBreakdownDonutProps) {
+  const { t } = useTranslation("dashboard");
+
+  if (Object.keys(data.valueBreakdown).length === 0) {
+    return null;
+  }
+
+  // Exclude zero-value types to prevent visual clutter in donut chart
+  const chartData = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)